.github/workflows/codeql-analysis.yml

```diff
@@ -19,11 +19,6 @@ jobs:
         # a pull request then we can checkout the head.
         fetch-depth: 2
```

Comment:
You can remove this `with` if we are not doing `HEAD^2` (that is outside the range I can suggest).

Reply:
Thanks. Fixed.